frontend/packages/component-library/src/video/Video.tsx

```diff
@@ -94,6 +123,30 @@ const Video: React.FC<VideoProps> = ({
             </BodyThreeText>
           </div>
         );
+      case 'cookie-blocked':
+        return (
+          <div className={classNames(moduleStyles.errorPlaceholder)}>
+            <FontAwesomeV6Icon
+              iconName="exclamation-circle"
+              iconStyle="solid"
+            />
+            <BodyTwoText>
+              <StrongText>{errorHeading || 'Video unavailable'}</StrongText>
+            </BodyTwoText>
+            <BodyThreeText>
+              {errorBody ||
+                'Please enable Functional Cookies and refresh the page to play this video.'}
+            </BodyThreeText>
+
+            <Button
+              onClick={() => {
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                (window as any).OneTrust.ToggleInfoDisplay();
+              }}
+              text={'Cookie Settings'}
+            />
+          </div>
```

Comment:
Can you add styles for the button so there's a `1.5rem` gap between it and the "Please enable…" text? I can also take a look at this after this is merged if you prefer!